Microsoft.Identity.Web/Client/ITokenAcquisition.cs

```diff
@@ -44,64 +47,9 @@ public interface ITokenAcquisition
         /// </summary>
         /// <param name="context">HttpContext associated with the Controller or auth operation</param>
         /// <param name="scopes">Scopes to request for the downstream API to call</param>
-        /// <param name="tenantId">Enables to override the tenant/account for the same identity. This is useful in the 
```

Comment:
So, do I understand that the suggestion is to not subscribe to the token validated event?

Reply:
correct. just need to validate the incoming access token. 

---
In reply to: [318912645](https://github.com/Azure-Samples/active-directory-dotnet-native-aspnetcore-v2/pull/93#discussion_r318912645) [](ancestors = 318912645)